pa11y/.gitignore

```diff
@@ -1,2 +1 @@
-build
 screenshots
```

Comment:
why this?

Reply:
Good question.
It is because the only thing that the P11y directory is "managing"/containing is the screenshots directory. When the Pa11y cmd is executed it will generate screenshots for the configured URL's - and they will end up in /pa11y/screenshots